WMFComponents/Sources/WMFComponents/Components/Profile/WMFProfileViewModel.swift

```diff
@@ -6,16 +6,18 @@ public class WMFProfileViewModel: ObservableObject {
     let isLoggedIn: Bool
     let localizedStrings: LocalizedStrings
     let inboxCount: Int
+    private weak var coordinatorDelegate: ProfileCoordinatorDelegate?
 
-    public init(isLoggedIn: Bool, localizedStrings: LocalizedStrings, inboxCount: Int) {
+    public init(isLoggedIn: Bool, localizedStrings: LocalizedStrings, inboxCount: Int, coordinatorDelegate: ProfileCoordinatorDelegate?) {
         self.isLoggedIn = isLoggedIn
         self.localizedStrings = localizedStrings
         self.inboxCount = inboxCount
+        self.coordinatorDelegate = coordinatorDelegate
         loadProfileSections()
     }
 
     private func loadProfileSections() {
-        profileSections = ProfileState.sections(isLoggedIn: isLoggedIn, localizedStrings: localizedStrings, inboxCount: inboxCount)
+        profileSections = ProfileState.sections(isLoggedIn: isLoggedIn, localizedStrings: localizedStrings, coordinatorDelegate: coordinatorDelegate)
```

Comment:
This inbox count parameter might be necessary, it caused a bug for me earlier without it.

Reply:
No idea why I removed it, fixed!